### PR TITLE
Take user to space on click space name

### LINKED ISSUE #1225: Clicking on the space name in Timelineproposal should take user to space
When clicking on the space name in `TimelineProposal` and `TimelineProposalPreview` it should take the user to the space and not to the proposal. 
image

### PR DESCRIPTION
**Pull request by sasicodes**
Fixes #1225 

Changes proposed in this pull request:
- On clicking on the space name in `TimelineProposal` and `TimelineProposalPreview` it will take the user to the space.

Reference:

https://www.loom.com/share/73a74c2c04cd4a2e9971e5dcc7171cf2

### CODE REVIEW

**Comment by samuveth** [comment]
@sasicodes when clicking on the space image we should also go to the space. Text should be white when hoving the image or the text, you can use tailwind group for that. https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state

**Comment by sasicodes** [comment]
@samuveth yeah, i can do that

**Comment by sasicodes** [comment]
@samuveth https://www.loom.com/share/56f306eab4de48778fc972a359836d1d

**Comment by sasicodes** [comment]
@mktcode In that case, shall we remove the link in `TimelineProposal`?

**Comment by sasicodes** [comment]
@mktcode https://www.loom.com/share/b42f474452004bcc8f0a61ab3509da70

**Comment by mktcode** [comment]
Great. Btw, there's always a test build for each pull request.

image

So you don't need to record so much. ;)

**Review by samuveth — APPROVED** [review]
Works well!

File: src/components/TimelineProposal.vue [hunk 1]
```diff
@@ -39,8 +39,20 @@ watchEffect(() => {
     >
       <div>
         <div class="mb-2">
-          <Token :space="proposal.space" size="28" />
-          <span class="ml-2" v-text="proposal.space.name" />
+          <router-link
```
Comment (bonustrack) [inline src/components/TimelineProposal.vue:42]: Adding link is not relevant here because this component is only used when we are in a space page

Comment (sasicodes) [reply src/components/TimelineProposal.vue:42]: Right, this `TimelineProposal` is used on this page alone. Will remove here

**Pull request merged**